imageatm/components/data_prep.py

```diff
@@ -77,7 +76,7 @@ def __init__(
         """
         self.job_dir = Path(job_dir).resolve()
         if not self.job_dir.exists():
```

Comment:
We could make use of `exist_ok`
```suggestion
            self.job_dir.mkdir(parents=True, exist_ok=True)
```